api/requirements.txt

```diff
@@ -8,3 +8,4 @@ requests~=2.21.0
 uwsgi~=2.0.0
 django-nocaptcha-recaptcha==0.0.20  # updated manually
 djangorestframework-bulk~=0.2.0
+coverage~=4.5.3
```

Comment:
Is there a way to only have such dependencies when testing, but not in the final build?

This does not need to be solved now, but if we think about hardening later, we should make sure that stuff not used at runtime does not get built into the image.

Reply:
It's possible, but it introduces additional complexity (such as waiting for coverage to be installed with every test run or having a second requirements.txt file). I believe as of now, the disadvantages outweigh the advantages. (Also considering that we do not even install the latest version of djangorestframework and use the abandoned django-nocaptcha-recaptcha and djangorestframework-bulk)